lib/cmds/start.ts

```diff
@@ -100,6 +107,13 @@ function start(options: Options) {
         '-Dwebdriver.ie.driver=' +
         path.join(outputDir, binaries[IEDriver.id].executableFilename(osType)));
   }
+  if (downloadedBinaries[AndroidSDK.id] != null) {
```

Comment:
Could we also hide this behind an `--appium` flag? A user might have the binaries but might not want to start an Android emulator every time.